packages/design/src/Form/components/DateOfBirth/DateOfBirth.tsx

```diff
@@ -36,20 +37,33 @@ export const DateOfBirthPattern: PatternComponent<DateOfBirthProps> = ({
         {required && <span className="required-indicator">*</span>}
       </legend>
       {hint && (
-        <span className="usa-hint" id="mdHint">
+        <span className="usa-hint" id={`hint-${monthId}`}>
           {hint}
         </span>
       )}
+      {error && (
+        <div
+          className="usa-error-message"
+          id={`input-error-message-${monthId}`}
+          role="alert"
+        >
+          {error.message}
+        </div>
+      )}
       <div className="usa-memorable-date">
         <div className="usa-form-group usa-form-group--month usa-form-group--select">
           <label className="usa-label" htmlFor={monthId}>
             Month
           </label>
           <select
-            className="usa-select"
+            className={classNames('usa-input', {
+              'usa-input--error': error,
+            })}
             id={monthId}
             {...register(monthId)}
-            aria-describedby="mdHint"
+            aria-describedby={
+              error ? `${monthId} input-error-message-${monthId}}` : monthId
```

Comment:
I left a comment for you on your other PR about the logic for the `aria-describedby` attribute. The same comment is true here where I think you'll want to have 4 branches of a conditional - error only, hint only, both error and hint, neither error nor hint.